tests/common/dualtor/data_plane_utils.py

```diff
@@ -85,28 +94,38 @@ def send_t1_to_server_with_action(ptfhost, ptfadapter, tbinfo):
     arp_setup(ptfhost)
     
     duthosts = []
-    def t1_to_server_io_test(activehost, standbyhost=None, tor_port=None, delay=0, action=None, verify=False):
+    def t1_to_server_io_test(activehost, standbyhost=None, tor_port=None,
+                            delay=0, action=None, verify=False,
+                            downstream_dst_ip=None):
```

Comment:
Could we change this to take the PTF/ToR port name instead? That way we can put less demand on the tests themselves since they will already be using these names to interact with other fixtures.

Reply:
Sure, any change is good which makes testcase look simple. Are you referring to changing `tor_port` to use string name, instead of integer?

Reply:
Addressed